packages/Disclosure/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@igloo-ui/disclosure",
+  "version": "0.0.0",
+  "main": "dist/Disclosure.js",
+  "module": "dist/Disclosure.js",
+  "types": "dist/Disclosure.d.ts",
+  "styles": "dist/disclosure.css",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "git+https://github.com/gsoft-inc/ov-igloo-ui.git",
+  "author": "Officevibe Inc.",
+  "contributors": [
+    "Vicky Comeau vicky.comeau@gsoft.com"
+  ],
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "react": ">= 16.8.6"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup -c rollup.config.js"
+  },
+  "dependencies": {
+    "@igloo-ui/tokens": "^2.0.0",
+    "classnames": "^2.3.2",
+    "framer-motion": "^6.5.1",
```

Comment:
Is there a reason why we're using framer-motion instead of react-spring here?

Reply:
Yes, framer motion had an easy built in way to animate to auto height. I was not able to do this with react-spring. I've been running into a lot of little bugs with react-spring as well so I tried something different.